wagtail/contrib/modeladmin/templates/modeladmin/create.html

```diff
@@ -21,6 +21,19 @@
 
     {{ view.media.js }}
     {% prepopulated_slugs %}
+
+    <script>
+        $(function() {
+            $('[data-wagtail-tooltip]').tooltip({
+                animation: false,
+                title: function() {
+                    return $(this).attr('data-wagtail-tooltip');
```

Comment:
I do wonder whether we should rely be relying on js to fill in tooltip attributes for accessibility - possibly we should have them there in the DOM (and not in a data-attribute) before the js loads as well. But I think that's a discussion for another time!